modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -6,19 +6,33 @@ endif::[]
 
 image::manage-service-mesh-breadcrumb.png[75%, 75%]
 
-After you bind your API, the API in API Manager is marked as Active.You are now ready to apply policies to protect your APIs.
+After you configure Service Mesh, you are now ready to manage your Mule and non-Mule applications from API Manager. You can apply policies to ensure various levels of security for your microservices. 
```

Comment:
After you configured Service Mesh, you are now ready to manage your non-Mule microservcies from API Manager

Reply:
Done!